tests/sparseml/onnx/benchmark/test_info.py

```diff
@@ -64,151 +62,27 @@ def cpu_runner_fixture(
 
 class TestLoadModel:
     def test_load_model_from_sparsezoo_model(self, mobilenet_fixture: Model):
-        onnx_model = onnx.load(mobilenet_fixture.onnx_file.path)
+        onnx_model = onnx.load(mobilenet_fixture.onnx_model.path)
         assert load_model(mobilenet_fixture) == onnx_model
 
     def test_load_model_from_sparsezoo_file(self, mobilenet_fixture: Model):
-        onnx_model = onnx.load(mobilenet_fixture.onnx_file.path)
-        assert load_model(mobilenet_fixture.onnx_file) == onnx_model
+        onnx_model = onnx.load(mobilenet_fixture.onnx_model.path)
+        assert load_model(mobilenet_fixture.onnx_model) == onnx_model
 
     def test_load_model_from_stub(self, mobilenet_fixture: Model):
-        onnx_model = onnx.load(mobilenet_fixture.onnx_file.path)
-        assert (
-            load_model(
-                TEST_STUB, override_parent_path=mobilenet_fixture.override_parent_path
-            )
-            == onnx_model
-        )
+        onnx_model = onnx.load(mobilenet_fixture.onnx_model.path)
+        assert load_model(TEST_STUB, path=mobilenet_fixture._path) == onnx_model
 
     def test_load_model_from_path(self, mobilenet_fixture: Model):
-        onnx_model = onnx.load(mobilenet_fixture.onnx_file.path)
-        assert load_model(mobilenet_fixture.onnx_file.path) == onnx_model
+        onnx_model = onnx.load(mobilenet_fixture.onnx_model.path)
+        assert load_model(mobilenet_fixture.onnx_model.path) == onnx_model
 
     def test_load_model_from_onnx(self, mobilenet_fixture: Model):
-        mobilenet_fixture.onnx_file.download()
-        onnx_model = onnx.load(mobilenet_fixture.onnx_file.path)
+        onnx_model = onnx.load(mobilenet_fixture.onnx_model.path)
 
         assert load_model(onnx_model) == onnx_model
 
 
-class TestLoadData:
```

Comment:
goes along with the comment from removing the zoo data utils, we should avoid getting rid of this if possible

Reply:
Ok, this is the one you mentioned in https://github.com/neuralmagic/sparsezoo/pull/179. Will bring those back.